apps/reward-manager/src/config/settings.json

```diff
@@ -0,0 +1,15 @@
+{
+    "mainnet": {
+        "poolIds": {
+            "0xba100000625a3754423978a60c9317c58a424e3d": "0x5c6ee304399dbdb9c8ef030ab642b10820db8f56000200000000000000000014",
```

Comment:
If these configs are not used in the sNOTE reinvestment then we can delete the file.